README.md

```diff
@@ -1,17 +1,24 @@
 [![Runbot Status](https://runbot.odoo-community.org/runbot/badge/flat/122/12.0.svg)](https://runbot.odoo-community.org/runbot/repo/github-com-oca-l10n-italy-122)
-[![Build Status](https://travis-ci.org/OCA/l10n-italy.svg?branch=12.0)](https://travis-ci.org/OCA/l10n-italy)
+[![Build Status](https://travis-ci.org/OCA/l10n-italy.svg?branch=12.0)](https://travis-ci.org/OCA/l10n-italy/branches)
 [![Coverage Status](https://coveralls.io/repos/OCA/l10n-italy/badge.svg?branch=12.0)](https://coveralls.io/r/OCA/l10n-italy?branch=12.0)
 [![Code Climate](https://codeclimate.com/github/OCA/l10n-italy/badges/gpa.svg)](https://codeclimate.com/github/OCA/l10n-italy)
 
 Odoo Italia Modules
 ===================
 
+Contributing
+------------
+Do you want to contribute? Please read our [contributing guidelines](https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rsthttps://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md).
```

Comment:
There's a missing space between `CONTRIBUTING.rst` and `https`.

Reply:
oops, wrong paste. Thanks, fixed